0030-update-threshold-cryptography.md

```diff
@@ -0,0 +1,79 @@
+# HIP 30: Switch to BLS12-381 for Threshold Cryptography
+
+- Author(s): [@vihu](http://github.com/vihu), [@Vagabond](https://github.com/vagabond/)
+- Start Date: 2021-04-19
+- Category: Technical
+- Original HIP PR: TBD
+- Tracking Issue: TBD
+
+# Summary
+[summary]: #summary
+
+Helium [Distributed Key Generation](https://github.com/helium/erlang-dkg) and [Honeybadger Consensus Protocol](https://github.com/helium/erlang-hbbft) both rely on curve SS512 for pairing based cryptography.
```

Comment:
```suggestion
Helium [Distributed Key Generation](https://github.com/helium/erlang-dkg) and [Honeybadger Consensus Protocol](https://github.com/helium/erlang-hbbft) both rely on curve SS512 for pairing based cryptography.

Curve SS512 is a very old curve and is not commonly used anymore. In addition the library we used to do pairing based cryptography, Ben Lynn's `pbc` library, has not seen major maintenance since 2013.
```